keyboards/ckeys/handwire_101/handwire_101.h

```diff
@@ -0,0 +1,23 @@
+#ifndef HANDWIRE_101_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`?

It's simpler, and less prone to user error, so if you could make this switch?

```suggestion
#pragma once
```